examples/boards/ppro/blinky.py

```diff
@@ -0,0 +1,71 @@
+
```

Comment:
Following the python philosophy of explicit the directory name of `papilio_pro` is preferred.